jsk_apc2016_common/samples/README.md

```diff
@@ -0,0 +1,31 @@
+samples
+=======
+
+**samples** includes files used for picking an APC object from table by Fetch.
+
+
+Usage
+-----
+
+```
+$ roslaunch sample_fetch_start.launch
+$ ./sample-fetch-pick.l
```

Comment:
I used trial3 for the demonstration.
I will change the function name  from trial3 to pick at next commit.